README.md

```diff
@@ -126,6 +126,7 @@ try to submit a small toy example already available.
 
 ```bash
 MY_API_KEY="xxxxx"
+CLOUDOS="https://cloudos.lifebit.ai"
 WORKSPACE_ID="5c6d3e9bd954e800b23f8c62"
```

Comment:
```Suggestion
WORKSPACE_ID="5xxxxxxxxxxxxxxxxxxxx"
```

No need to add the specific workspace ID, even though it is not a secret variable.